src/components/molecules/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Select from './Select';
+import { ISelectInput } from './SelectInput.model';
+
+const SelectInput = ({
+  isMulti,
+  options,
+  handleOnClick,
+  selectCaption,
+  inputPlaceholder,
+}: ISelectInput): JSX.Element => {
+  const manageIsMultiState = () => {
```

Comment:
Do we really need this function? We can use bang operator `!!` in line 21 instead of this